### PR TITLE
reword, typos, grammar, etc

### PR DESCRIPTION
**Pull request by gaucheph**
- fix some typos, grammar
- consistent casing for Elasticsearch
- reword some phrases to be more active

### CODE REVIEW

**Comment by gaucheph** [comment]
hey @fruch! i like the project and i think it's tremendously valuable. i noticed some typos in the readme and started to fix them but then also "fixed" a few other things. let me know what you think :)

**Comment by fruch** [comment]
> hey @fruch! i like the project and i think it's tremendously valuable. i noticed some typos in the readme and started to fix them but then also "fixed" a few other things. let me know what you think :)

Hi @gaucheph I really happy you find it helpful, I'll surely going to merge those.

Also it would be nice to hear a bit more about the use case you have for it, since just yesterday I've got a CFP accepted to talk about this project (in https://pycon.org.il), and it's a great timing to have some written "testimonial" (snapshot of graphs or dashboards, would be a great bonus)

File: README.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ A plugin to send pytest test results to [ELK] stack, with extra context data
...
+* Report each test result into Elasticsearch as they finish
+* Automatically append contextual data to each test:
   * git information such as `branch` or `last commit` and more
   * all of CI env variables
     * Jenkins
     * Travis
     * Circle CI
     * Github Actions
   * username if available
-* Report a test summery to Elastic for each session with all the context data
```
Comment (fruch) [inline README.md:23]: yep, the summer is coming :+1:

**Review by fruch — APPROVED** [review]
LGTM, and you just got hired as a technical writer for this project. LOL.

**Pull request merged**